vanilla-framework/vanilla-components
# Add Jest testing to components

## Done

- Added Jest tests to check for each component and that it renders without error.

## QA

 -Pull code
- Run `npm run test`
- Type `a` to run all tests
- All three tests should pass like so:

<img width="346" alt="screen shot 2017-10-12 at 13 35 30" src="https://user-images.githubusercontent.com/505570/31496483-71cbe122-af53-11e7-8dad-a67ce9ae27f5.png">

Fixes #3, Fixes #4
